groups/groups.yaml

```diff
@@ -22,6 +22,83 @@ groups:
       - codyclark@google.com
       - vonguard@gmail.com
 
+  - email-id: chairs-and-techleads@kubernetes.io
+    name: chairs-and-techleads
+    description: |-
+      SIG/WG/UG Chairs and TLs
+    settings:
+      ReconcileMembers: "true"
+    owners:
+      - contributors@kubernetes.io
```

Comment:
Why does this group need owners? I suspect the answer is "moderation" but it would be helpful to have it documented somehow.

Why is contributors@kubernetes.io not defined in this file?

Reply:
> Why does this group need owners? I suspect the answer is "moderation" but it would be helpful to have it documented somehow.

I'd be ok with removing it if we don't allow everyone to post. I'm not sure if there are other moderation powers that might be useful here.

> Why is contributors@kubernetes.io not defined in this file?

No idea. contributors@ are current owners of the google group.

@parispittman @mrbobbytables @idvoretskyi @jdumars @castrojo (folks from https://github.com/kubernetes/community/issues/4673#issuecomment-623556960) - can we get contributors@ into this file?

Reply:
This is probably blowing scope... what's the difference between community@ and contributors@? I think this was mentioned somewhere/in some meeting but I can't remember :see_no_evil:

Reply:
community is a group account that acts as the point of contact for things like events etc and is shared with a larger group of people. It cannot own docs or other things as it's not an actual user. contributors@ is an actual user account, not a group. It can own docs/drive and a bunch of other things 👍

Reply:
@mrbobbytables Thanks for the context! Three questions:

1. Given that no account needs to own this gsuite email anymore, would it be ok to remove contributors@ from the list?

2. Would it be useful to have community@ here as as member to help with posting information about events, etc?

3. Is contributors@ and the list of people behind that alias documented/can we document it?

Reply:
> 1. Given that no account needs to own this gsuite email anymore, would it be ok to remove contributors@ from the list?


I would still leave it in there for moderation purposes. It's the "backup" method we use in case we have to step in.


> 2. Would it be useful to have community@ here as as member to help with posting information about events, etc?


I think that's a good idea 👍 


> 3. Is contributors@ and the list of people behind that alias documented/can we document it?

I don't think it's well documented at this time. >_> As a start, the folks who have access to it could be listed in our [moderation doc](https://github.com/kubernetes/community/blob/master/communication/moderators.md).

Reply:
>> 2. Would it be useful to have community@ here as as member to help with posting information about events, etc?

> I think that's a good idea +1

Added